spec/components/create_a_job_sidebar_component_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe CreateAJobSidebarComponent, type: :component do
+  let(:vacancy) { create(:vacancy, completed_step: completed_step) }
+  let(:completed_step) { 0 }
+  let(:current_step) { 1 }
+  let(:school_group_user?) { true }
+
+  before do
+    allow_any_instance_of(HiringStaff::JobCreationHelper).to receive(:current_step).and_return(current_step)
+    allow_any_instance_of(HiringStaff::JobCreationHelper).to receive(:school_group_user?).and_return(school_group_user?)
+    @result = render_inline(described_class.new(vacancy: vacancy))
```

Comment:
This should be

`let(:inline_result) { render_inline(described_class.new(vacancy: vacancy)) }`

Reply:
We need it to be rendered before each block, but after lines 10/11. Not sure how we can achieve that other than this?

Reply:
ah right, cannot think of a better way for now